docker/Dockerfile.frontend

```diff
@@ -1,6 +1,5 @@
-FROM node:12.2.0-alpine
+FROM node:13.7.0-alpine
```

Comment:
Consider using `node:latest` or sth like that

Reply:
Usually it is safer to freeze image version to ensure nothing breaks on major releases of underlying image. I would only use `latest` tag on images that I manage myself or images I do not care if they break.